package.json

```diff
@@ -16,19 +16,29 @@
     "html5",
     "manifest"
   ],
-  "author": "Scott Hillman",
+  "author": {
+    "name": "Scott Hillman"
+  },
   "license": "MIT",
   "readmeFilename": "README.md",
   "gitHead": "58e17b7e8757385a1988e30735c46c86e78a2aba",
   "devDependencies": {
-    "should": "~2.1.1",
-    "mocha": "~1.16.2"
+    "event-stream": "^3.1.7",
+    "mocha": "~1.16.2",
+    "should": "~2.1.1"
   },
   "dependencies": {
-    "event-stream": "~3.0.20",
     "through": "~2.3.4",
     "gulp-util": "~2.2.6",
     "slash": "^0.1.1",
     "minimatch": "~2.0.1"
-  }
+  },
+  "bugs": {
+    "url": "https://github.com/hillmanov/gulp-manifest/issues"
+  },
+  "homepage": "https://github.com/hillmanov/gulp-manifest",
+  "_id": "gulp-manifest@0.0.5",
```

Comment:
Not sure where the rest of the meta data is coming from, but some of it isn't even correct. The npm version is now at 0.0.7, and will very soon be bumped. The shasum will be changed all the time, and the resolved thing will be too difficult to maintain. Can we drop this info as well?